src/components/Tranzactii.js

```diff
@@ -74,6 +75,10 @@ const Tranzactii = () => {
             onChange={handleChange}
             required
           />
+          <select name="transaction_type" value={formData.transaction_type} onChange={handleChange} required>
+            <option value="Intrare">Intrare</option>
+            <option value="Iesire">Iesire</option>
```

Comment:
These can also be presented in English, while the values remain in Romanian for now.